constellation/__init__.py

```diff
@@ -7,9 +7,9 @@
     ImageReference
 
 __all__ = [
-    Constellation,
-    ConstellationContainer,
-    ConstellationService,
-    ConstellationMount,
-    ImageReference
+    "Constellation",
```

Comment:
?

Reply:
Python started wanting these to be quoted. I have no idea what triggered it :cry: